Cesium.CodeGen/Ir/BlockItems/FunctionDeclaration.cs

```diff
@@ -0,0 +1,82 @@
+using Cesium.CodeGen.Contexts;
+using Cesium.CodeGen.Contexts.Meta;
+using Cesium.CodeGen.Extensions;
+using Cesium.CodeGen.Ir.Types;
+using Cesium.Core;
+
+namespace Cesium.CodeGen.Ir.BlockItems;
+
+internal class FunctionDeclaration : IBlockItem
+{
+    private readonly string identifier;
+    private readonly FunctionType functionType;
+    private readonly string? cliImportMemberName;
```

Comment:
Can we have editorconfig which would suggest these things? I always work on defaults and forget to pay attention to this things.

Reply:
Borrow configuration from https://stackoverflow.com/a/49428349/354473

Reply:
> Can we have editorconfig which would suggest these things?

We sure can, but I've no idea how to set it up. I'd gladly accept a contribution :)

> I always work on defaults

Me too, but I guess we have different sets of defaults :)

Reply:
> I'll gladly accept a contribution :)

Too late, I already done that yesterday because I decide that better pollute this PR then have this issue come up over and over again

> we have different sets of defaults :)

Sure! VS and Rider obviously has different set of default 😄